content/en/user-guide/aws/dms/index.md

```diff
@@ -120,10 +120,12 @@ DMS is in a preview state on LocalStack and only supports some selected use case
 | RDS MariaDB        | Kinesis     | full-load, cdc  |
 | RDS MySQL          | Kinesis     | full-load, cdc  |
 | S3                 | Kinesis     | full-load, cdc  |
+| Aurora PostgreSQL  | Kinesis     | full-load, cdc  |
+| RDS PostgreSQL     | Kinesis     | full-load, cdc  |
```

Comment:
```suggestion
| Aurora PostgreSQL  | Kinesis     | full-load, cdc  |
| RDS PostgreSQL     | Kinesis     | full-load, cdc  |
| PostgreSQL (external) | Kinesis     | full-load, cdc  |
```

You'll probably need to align the table again 😅 as PostgreSQL external is pretty long. Not sure if the linter does that